packages/gatsby-link/src/__tests__/index.js

```diff
@@ -86,9 +86,10 @@ describe(`<Link />`, () => {
     }
 
     it(`accepts to as a string`, () => {
+      jest.spyOn(global.console, `warn`)
```

Comment:
Nit: but I'd keep this test separate from this functionality.

In other words--I'd probably add a test testing that `does not warn when internal link` which just tests the console.warn functionality.

Reply:
Heh yeah my bored personnality took over :D Fixed